lib/api/route/messages.dart

```diff
@@ -258,6 +258,39 @@ class SendMessageResult {
   Map<String, dynamic> toJson() => _$SendMessageResultToJson(this);
 }
 
+/// https://zulip.com/api/update-message
+Future<UpdateMessageResult> updateMessage(
+  ApiConnection connection, {
+  required int messageId,
+  String? topic,
```

Comment:
```suggestion
  TopicName? topic,
```

:slightly_smiling_face: